src/app.ts

```diff
@@ -4,7 +4,7 @@ import mongoose from 'mongoose';
 import authRouter from './route/auth.route';
 
 const app = express();
-const port = process.env.PORT ?? 3000;
+const port = process.env.PORT || 8000;
```

Comment:
```suggestion
const port = process.env.PORT ?? 8000;
```

Recommend to use `??` when dealing with env variables.

For example, if port is on 0:
```
console.log(0 ?? 3000); // 0
console.log(0 || 3000); // 3000
```

See:
1. [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing#description)
2. [Stack overflow discussion](https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or)

Reply:
if .env file write PORT=
will throw new error

Reply:
```
console.log(undefined || "not found") // "not found"
console.log(null      || "not found") // "not found"

console.log(undefined ?? "not found") // "not found"
console.log(null      ?? "not found") // "not found"
```

Reply:
because in .env you write PORT=
system will get a empty string ''
not null or undefined

Reply:
Using nullish is a stricter approach that allows you to quickly identify imply problems. 

For your example, if process.env.PORT defaults to an empty string (""), and you deploy to the cloud, you would immediately know that process.env.PORT has not been set. 

But if you use the logical OR operator (||), it won't throw any error and will use the default value, leading to hidden problem, which wrong port setting on cloud.

Reply:
remove as discussed